controllers/databasecluster_controller.go

```diff
@@ -906,6 +934,11 @@ func (r *DatabaseClusterReconciler) genPXCBackupSpec(
 				Region:            backupStorage.Spec.Region,
 				EndpointURL:       backupStorage.Spec.EndpointURL,
 			}
+		case everestv1alpha1.BackupStorageTypeAzure:
+			storages[backup.Spec.BackupStorageName].Azure = &pxcv1.BackupStorageAzureSpec{
+				ContainerPath:     backupStorage.Spec.Bucket,
```

Comment:
What about gcs support for PXC?

Reply:
Per our discussion, PXC does not have GCS support yet.